#
# the examples need libpcl, so we remove the exmaples dep from the all target
#
# make -C examples
# make[2]: Entering directory
# `/home/milan/ptxdist/ptxdist-testing/build/epoll-lib-0.10/examples'
# powerpc-405-linux-gnu-gcc -g -I- -I. -I../include -I/usr/src/linux/include
# -DUNIX -D_DEBUG -DHTTP_BLASTER -o ../bin/epoll-blaster ./epoll-test.c
# -L../lib -lepoll -lpcl
# epoll-test.c:61:17: pcl.h: No such file or directory
# epoll-test.c:85: syntax error before "coroutine_t"
#
--- epoll-lib-0.10-orig/Makefile	Sat Sep  6 01:38:08 2003
+++ epoll-lib-0.10/Makefile	Wed Nov  5 00:35:50 2003
@@ -31,9 +31,9 @@
 MAKE = make
 
 ifeq ($(EPOLLCFG), release)
-CFLAGS = -O3 $(INCLUDE) -DUNIX
+CFLAGS = -O3 $(INCLUDE) -DUNIX $(XCFLAGS)
 else
-CFLAGS = -g $(INCLUDE) -DUNIX -D_DEBUG
+CFLAGS = -g $(INCLUDE) -DUNIX -D_DEBUG $(XCFLAGS)
 endif
 
 SOURCES = $(SRCDIR)/epoll.c
@@ -46,7 +46,7 @@
 $(OUTDIR)/%.o: $(SRCDIR)/%.c
 	$(CC) $(CFLAGS) -o $(OUTDIR)/$*.o -c $(SRCDIR)/$*.c
 
-all: .depend $(TARGET) epoll-example
+all: .depend $(TARGET)
 
 .depend: $(SOURCES)
 	$(MKDEP) $(CFLAGS) $(SOURCES)
